config/routes.rb

```diff
@@ -1,3 +1,15 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  defaults format: 'json' do
+    namespace :api do
+      namespace :v1 do
```

Comment:
Not sure of the best approach there. However, I'd prefer to omit the `api` part, to be on a subdomain. I think keeping the API versions visible and explicit would help prevent sleepless nights